custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -330,11 +330,26 @@ def _reg_arg_docker(*parsers):
                 required=False,
                 help="Docker image to use to run {} in the {} mode, "
                 "or a directory, containing a Dockerfile, "
-                "which can be built into a docker image. ".format(
-                    ArgumentsOptions.MAIN_COMMAND, prog_name_lst[1]
+                "which can be built into a docker image. "
+                "If code dir contains requirements.txt file, DRUM tries to install dependencies during image build. "
+                "For Python/R models only. Use {} to skip installation.".format(
+                    ArgumentsOptions.MAIN_COMMAND,
+                    prog_name_lst[1],
+                    ArgumentsOptions.SKIP_DEPS_INSTALL,
                 ),
             )
 
+    @staticmethod
+    def _reg_arg_skip_deps_install(*parsers):
+        for parser in parsers:
+            parser.add_argument(
+                ArgumentsOptions.SKIP_DEPS_INSTALL,
+                default=False,
+                action="store_true",
+                required=False,
+                help="Skip dependencies installation during image build.",
```

Comment:
You probably need more description here. I would probably rather have a default off flag that is something like `--install-local-requirements` instead of one flag with lots of functionality and then another to turn half of it off.

Reply:
I decided to go with `--skip-deps-install` to keep default behavior as easy as simple. Say you have several models in tests and you don't need to remember to provide the flag or not to.

In case dependency installation fails for the reason we don't handle (dep is taken from some weird repo), user has the change to change Dockerfile and skip installation in DRUM.

Reply:
but if you're adding requirements in the UI, you know that you're adding requirements that will be installed on top your image, so you're already knowing this is going on.

Reply:
anyways, what happens if skip is flagged on but docker isn't engaged? Is that dealt with nicely?

Reply:
I guess its fine though having this skip functionality.

Reply:
yes, 
this flag is used only within the docker building

Reply:
you should say it will be ignored if the --docker flag is not engaged.

Reply:
in the description

Reply:
Added a check